code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -84,11 +84,29 @@
 	flags_inv = HIDEHAIR|HIDEFACIALHAIR
 	dog_fashion = null
 
-/obj/item/clothing/head/wizard/hood
+/obj/item/clothing/head/hooded/wizard
```

Comment:
You will probably need to add `clothing_flags = SNUG_FIT | CASTING_CLOTHES` here so that you can cast spells with your hood up, unless it is intentional that you cannot (I guess it would be somewhat of a small balance change to have the robes double as a replacement hat, but otherwise people won't ever use the hood)